Example/Podfile

```diff
@@ -9,6 +9,10 @@ use_frameworks!
 # Ignore the pod warnings.
 inhibit_all_warnings!
 
+target 'Example' do
```

Comment:
I created an issue where we can expand the Example project to contain all the requests with examples, etc... #18